visualizer/src/App.js

```diff
@@ -13,11 +14,14 @@ function App() {
   return (
     <Router>
       <Switch>
-        <Route path='/'>
+        <Route path='/project'>
           <ProjectContext project={service}>
             <Label />
           </ProjectContext>
         </Route>
+        <Route path='/'>
+          <Load />
+        </Route>
```

Comment:
Minor preference: I'd like it if the order was switched for clarity. (root path first)

Reply:
I think that changes the behavior

From the [React Router documentation](https://reactrouter.com/web/guides/quick-start)

> A <Switch> looks through its children <Route>s and renders the first one that matches the current URL.

so `/` would match everything. The examples on that page put `/` last